compress.c

```
@@ -28,8 +28,11 @@
    memory, Z_BUF_ERROR if there was not enough room in the output buffer,
    Z_STREAM_ERROR if the level parameter is invalid.
 */
-z_int32_t Z_EXPORT PREFIX(compress2)(unsigned char *dest, z_uintmax_t *destLen, const unsigned char *source,
-                        z_uintmax_t sourceLen, z_int32_t level) {
+#ifdef ZLIB_COMPAT
+int compress2_z(unsigned char *dest, z_size_t *destLen, const unsigned char *source, z_size_t sourceLen, int level) {
+#else
+int32_t zng_compress2(uint8_t *dest, size_t *destLen, const uint8_t *source, size_t sourceLen, int32_t level) {
```

Cppcheck (2.21.0)

[style] The function 'zng_compress2' should have static linkage since it is not used outside of its translation unit.

(staticFunction)